.prettierrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   printWidth: 100,
   tabWidth: 2,
   useTabs: false,
-  semi: true,
```

Comment:
In my private configuration, it is usually set to `semi: false`, what do you think @zengkaiwen @ohaaha